Yelp/Business.swift

```diff
@@ -85,10 +85,10 @@ class Business: NSObject {
     }
     
     class func searchWithTerm(_ term: String, completion: @escaping ([Business]?, Error?) -> Void) {
```

Comment:
this one could also be refactored 😛  but THIS IS ALREADY GREAT. :) @harley I'll let you merge